tap_github/streams.py

```diff
@@ -2,6 +2,7 @@
 
 from typing import Any, Dict, Iterable, List, Optional
 
+import requests
```

Comment:
This tells me you forgot to pull the latest `main` branch before writing your code ;)

Reply:
It's just there 3 lines above, no need to duplicate it :)

Reply:
ahh missed that for some reason haha. Thanks for the catch